pubsub/cloud-client/quickstart.py

```diff
@@ -19,76 +19,110 @@
 
 
 def end_to_end(project_id, topic_name, subscription_name, num_messages):
-    # [START pubsub_end_to_end]
+    # [START pubsub_quickstart_end2end_basic]
     import time
 
+    from google.api_core.exceptions import NotFound
     from google.cloud import pubsub_v1
 
     # TODO project_id = "Your Google Cloud Project ID"
     # TODO topic_name = "Your Pub/Sub topic name"
+    # TODO subscription_name = "Your Pub/Sub subscription name"
     # TODO num_messages = number of messages to test end-to-end
 
-    # Instantiates a publisher and subscriber client
+    # Instantiate a publisher and a subscriber client
     publisher = pubsub_v1.PublisherClient()
     subscriber = pubsub_v1.SubscriberClient()
 
     # The `topic_path` method creates a fully qualified identifier
```

Comment:
I don't think these comments are helpful -- they use a lot of space. A curious user could add print topic_path to see what that looks like.